public/app/core/components/OptionsUI/color.tsx

```diff
@@ -2,26 +2,33 @@ import { css } from '@emotion/css';
 import React from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { useTheme2, useStyles2, ColorPicker } from '@grafana/ui';
+import { useTheme2, useStyles2, ColorPicker, IconButton } from '@grafana/ui';
 import { ColorSwatch } from '@grafana/ui/src/components/ColorPicker/ColorSwatch';
 
-/**
- * @alpha
- * */
-export interface ColorValueEditorProps {
+export interface ColorValueEditorSettings {
+  placeholder?: string;
+  enableNamedColors?: boolean; // defaults to true
+  isClearable?: boolean; // defaults to false
```

Comment:
Docstrings here maybe?
```suggestion
  /** defaults to true */
  enableNamedColors?: boolean;
  /** defaults to false */
  isClearable?: boolean;
```

Reply:
I don't know to what extent these docstrings are used in core though